src/span.js

```diff
@@ -37,7 +41,12 @@ export default class Span {
     _peer: Endpoint;
     _logs: Array<LogData>;
     _tags: Array<Tag>;
+    static emptyBuffer: Buffer = new Buffer(8);
     static _baggageHeaderCache: any;
+    static _thrift = new Thrift({
+        source: fs.readFileSync(path.join(__dirname, './jaeger-idl/thrift/jaeger.thrift'), 'ascii'),
```

Comment:
is this file (from submodule) going to be available when the library is used as a dependency?